composer.json

```diff
@@ -280,7 +280,7 @@
         "google/shopping-merchant-conversions": "0.2.1",
         "google/shopping-merchant-datasources": "0.5.1",
         "google/shopping-merchant-inventories": "0.6.1",
-        "google/shopping-merchant-lfp": "0.3.2",
+        "google/shopping-merchant-lfp": "0.0.0",
```

Comment:
this needs to be reverted as well - it seems like you added this as a "new component" when it already exists.